docs/releases/1.11.txt

```diff
@@ -269,6 +269,10 @@ Tests
   :class:`~django.test.runner.DiscoverRunner` to make it easier to customize
   the :setting:`DEBUG` setting when running tests.
 
+* Added support for Python :meth:`python:unittest.TestCase.subTest`'s
+  when using the :option:`--parallel <test --parallel>` option to the
```

Comment:
I usually say "when using the test --parallel option." rather than linking the command separately.